apps/speed.c

```diff
@@ -502,17 +502,15 @@ static const char *evp_md_name = NULL;
 static char *evp_mac_ciphername = "aes-128-cbc";
 static char *evp_cmac_name = NULL;
 
-static EVP_MD *obtain_md(const char *name, int *fetched)
+static EVP_MD *obtain_md(const char *name)
```

Comment:
It looks to me like this function can be dropped entirely, and be replaced with a called of `opt_md()`